### ISSUE TITLE
senderReceiverTimeOffset should use remoteOutboundRtpStats.roundTripTime  

### ISSUE THREAD

**Issue by yuanchao0310** [w3c/webrtc-extensions#109]
in https://github.com/w3c/webrtc-extensions/blob/main/explainer.md#api-2-sendercapturetimeoffset 
`senderReceiverTimeOffset = remoteOutboundRtpStats.timestamp - (remoteOutboundRtpStats.remoteTimestamp + remoteInboundRtpStats.roundTripTime / 2);
`
if receiver is a recv only endpoint, it cannot get remoteInboundRtpStats.roundTripTime, because  [remoteInboundRtpStats.roundTripTime](https://www.w3.org/TR/webrtc-stats/#dom-rtcremoteinboundrtpstreamstats-roundtriptime) use SR and RR to get rtt time, that means the receiver must send RTP to remote peer.

so need use [remoteOutboundRtpStats.roundTripTime](https://www.w3.org/TR/webrtc-stats/#dom-rtcremoteoutboundrtpstreamstats-roundtriptime), which can be calculated by XR RTCP at receiver side.
after update:

> stats = peerconnection.getStats();
remoteOutboundRtpStats = getRequiredStats(stats, "remote-outbound-rtp");
senderReceiverTimeOffset = remoteOutboundRtpStats.timestamp - (remoteOutboundRtpStats.remoteTimestamp + remoteOutboundRtpStats.roundTripTime / 2)

**Comment by henbos** [comment]
Makes sense to me, and in terms of implementation, this [was implemented](https://chromiumdash.appspot.com/commit/2562cf0105d914a3cebfdef4d818e76ff658387b) in chromium/libwebrtc in M95. However it looks like it was only implemented for the audio case, not the video case. Anyway that's an implementation issue, not a spec issue.
